### PR TITLE
Add Amazon Aurora 

### PR DESCRIPTION
**Pull request by inv-jishnu**
I have updated the `SetupDatabaseForAWS.md` document by adding the Amazon Aurora section to it.

### CODE REVIEW

**Review by scalar-boney — COMMENTED** [review]
Left one suggestion.
I think you need to add a proper title for PR.
Thank you.

File: docs/SetupDatabaseForAWS.md [hunk 1]
```diff
@@ -21,3 +21,16 @@ Note:-
...
+
+### Steps
+
+* Follow this [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.CreateInstance.html) to create an Amazon Aurora cluster for MySQL/PostgreSQL.
+* (Optional) Follow this [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.Managing.Performance.html) to manage performance and scaling for Aurora DB clusters and DB instances.
+* (Optional) Configure `Enhanced Monitoring` for Amazon Aurora from the [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/USER_Monitoring.OS.Enabling.html), monitoring is enabled on the Amazon Aurora cluster by default.
+
+Note:-
+
+* Autoscaling can be enabled in Amazon Aurora if you have created the Amazon Aurora cluster with one primary instance and at least one Aurora Replica. Read more about it in the [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.Integrating.AutoScaling.html).
```
[the remark this answers is not in the document]
Comment (inv-jishnu) [reply docs/SetupDatabaseForAWS.md]: Thanks for the suggestion.
I have removed the section as it does not seem to inform anything specific related to Scalar DB/DL.

File: docs/SetupDatabaseForAWS.md [hunk 2]
```diff
@@ -21,3 +21,12 @@ Note:-
...
 * Follow this [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_CreateDBInstance.html) to create an AWS RDS MySQL/PostgreSQL/Oracle/SQL Server database instance.
 * (Optional) Configure advanced monitoring services with the [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Monitoring.html), monitoring is enabled by default on AWS RDS database instances.
+
+
+## Amazon Aurora for MySQL/PostgreSQL
+
+### Steps
+
+* Follow this [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.CreateInstance.html) to create an Amazon Aurora cluster for MySQL/PostgreSQL.
+* (Optional) Follow this [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.Managing.Performance.html) to manage performance and scaling for Aurora DB clusters and DB instances.
```
Comment (scalar-boney) [inline docs/SetupDatabaseForAWS.md:31]: Is there any specific reason for adding performance and scaling steps for Aurora?

**Review by scalar-boney — APPROVED** [review]
LGTM!

File: docs/SetupDatabaseForAWS.md [hunk 3]
```diff
@@ -21,3 +21,10 @@ Note:-
 
 * Follow this [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_CreateDBInstance.html) to create an AWS RDS MySQL/PostgreSQL/Oracle/SQL Server database instance.
 * (Optional) Configure advanced monitoring services with the [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Monitoring.html), monitoring is enabled by default on AWS RDS database instances.
+
+## Amazon Aurora for MySQL and PostgreSQL
```
Comment (feeblefakie) [inline docs/SetupDatabaseForAWS.md:25]:
```suggestion
## Amazon Aurora MySQL and Amazon Aurora PostgreSQL
```
It's not `for` MySQL. Just a MySQL-compatible Amazon Aurora.

File: docs/SetupDatabaseForAWS.md [hunk 4]
```diff
@@ -21,3 +21,10 @@ Note:-
...
+
+### Steps
+
+* Follow this [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.CreateInstance.html) to create an Amazon Aurora cluster for MySQL/PostgreSQL.
```
Comment (feeblefakie) [inline docs/SetupDatabaseForAWS.md:29]:
```suggestion
* Follow this [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.CreateInstance.html) to create Amazon Aurora MySQL or Amazon Aurora PostgreSQL.
```

**Review by feeblefakie — CHANGES_REQUESTED** [review]

File: docs/SetupDatabaseForAWS.md (hunk 3 above)
Comment (inv-jishnu) [reply docs/SetupDatabaseForAWS.md]: Understood. Thank you for the correction. 
I have updated the section title as suggested.

File: docs/SetupDatabaseForAWS.md (hunk 4 above)
Comment (inv-jishnu) [reply docs/SetupDatabaseForAWS.md]: Thank you for the feedback.
I have updated the sentence as suggested.

**Comment by inv-jishnu** [comment]
@feeblefakie san,

I have updated the document as suggested.
Could you please take a look at it again when you get a chance?

**Review by feeblefakie — APPROVED** [review]

**Pull request merged**